src/renderers/dom/client/ReactBrowserEventEmitter.js

```diff
@@ -345,6 +345,19 @@ var ReactBrowserEventEmitter = Object.assign({}, ReactEventEmitterMixin, {
   },
 
   /**
+   * Protect against document.createEvent() returning null
+   * Some popup blocker extensions appear to do this:
+   * https://github.com/facebook/react/issues/6887
+   */
+  testEventPageXY: function() {
```

Comment:
Maybe let’s call it `supportsEventPageXY()` so it’s clear what’s being returned.